projects/cassandra-node-monkey/Vagrantfile

```diff
@@ -0,0 +1,14 @@
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.box = "ubuntu/trusty64"
+
+  config.vm.define :database do |database|
```

Comment:
This is leftover from something old, lets name it better since we are starting fresh project (node-monkey-cassandra or something like it)